packages/design-system/src/components/Drawer/Drawer.tsx

```diff
@@ -80,6 +81,7 @@ export const Drawer = (props: DrawerProps) => {
             className="ds-c-drawer__close-button"
             size="small"
             onClick={props.onCloseClick}
+            variation={props.closeButtonVariation}
```

Comment:
Added ability for themes to specify what button style they want their close button to be. Satisfies the ghosty "X" button for mgov's design.